src/get-transaction-path.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = (transaction) ->
+  origin = transaction['origin']
+  delimiter = ":"
```

Comment:
@kuba-kubula Well, `#` is also valid in the identifier. http://docs.mson.apiary.io/#reference/users/user/view-a-user-and-his-